src/Simulation/Simulator.cpp

```diff
@@ -735,6 +735,7 @@ std::string Simulator::SimulatorRunCommand(SimulationType simulation,
       if (m_waveType == WaveformType::FST) {
         command += " -fst";
       }
+      if (!m_waveFile.empty()) command += " -dumpfile=" + m_waveFile;
```

Comment:
What's the default name for wavefile in our GUI ? It appears to be "icarus" which doesn't align with other simulator defaults, correct ?

Reply:
Based off the settings, I think it should have been "sim_rtl.fst"
We only support fst for icarus via GUI as per the simulator docs.
![image](https://github.com/os-fpga/FOEDAG/assets/108307503/640ce32f-c0b7-493f-a152-7858fe2e848f)

Reply:
this is issue with simulate command, I'll create another PR

Reply:
Here is fix: https://github.com/os-fpga/FOEDAG/pull/1138